test/resources/valid-trusts-registration-api-5mld-nontaxable.json

```diff
@@ -0,0 +1,332 @@
+            "countryOfResidence": "DE"
+          }
+        ]
+      },
+      "deceased": {
+        "name": {
+          "firstName": "John",
+          "middleName": "William",
+          "lastName": "O'Connor"
+        },
+        "dateOfBirth": "1956-02-12",
+        "dateOfDeath": "2016-01-01",
+        "identification": {
+          "nino": "KC456736A"
+        }
+      },
+      "leadTrustees": {
+        "leadTrusteeInd": {
+          "name": {
+            "firstName": "1234567890 QwErTyUiOp ,.(\/)&'- name",
+            "lastName": "1234567890 QwErTyUiOp ,.(\/)&'- name"
+          },
+          "dateOfBirth": "2015-03-20",
+          "identification": {
+            "nino": "SJ123456B"
+          },
+          "phoneNumber": "+123456789012345678",
+          "countryOfResidence": "DE",
+          "nationality": "DE",
+          "legallyIncapable": false
+        }
+      },
+      "trustees": [
+        {
+          "trusteeInd": {
+            "name": {
+              "firstName": "1234567890 QwErTyUiOp ,.(/)&'- name",
+              "lastName": "1234567890 QwErTyUiOp ,.(/)&'- name"
+            },
+            "identification": {
+              "passport": {
+                "number": "KSJDFKSDHF645654514TRUSTEEIND1",
+                "expirationDate": "2020-02-02",
+                "countryOfIssue": "DE"
+              },
+              "address": {
+                "line1": "1234567890 QwErTyUiOp ,.(/)&'- name",
+                "line2": "1234567890 QwErTyUiOp ,.(/)&'- name",
+                "line3": "1234567890 QwErTyUiOp ,.(/)&'- name",
+                "line4": "1234567890 QwErTyUiOp ,.(/)&'- name",
+                "postCode": "Z99 2YY",
+                "country": "GB"
+              }
+            },
+            "dateOfBirth": "1983-09-24",
+            "countryOfResidence": "DE",
+            "nationality": "DE",
+            "legallyIncapable": false
+          }
+        }
+      ],
+      "protectors": {
+        "protectorCompany": [
+          {
+            "name": "1234567890 QwErTyUiOp ,.(/)&'- name",
+            "identification": {
+              "address": {
+                "line1": "1234567890 QwErTyUiOp ,.(/)&'- name",
+                "line2": "1234567890 QwErTyUiOp ,.(/)&'- name",
+                "line3": "1234567890 QwErTyUiOp ,.(/)&'- name",
+                "line4": "1234567890 QwErTyUiOp ,.(/)&'- name",
+                "postCode": "Z99 2YY",
+                "country": "GB"
+              }
+            },
+            "countryOfResidence": "DE"
+          }
+        ]
+      },
+      "settlors": {
+        "settlor": [
+          {
+            "name": {
+              "firstName": "1234567890 QwErTyUiOp ,.(/)&'- name",
+              "lastName": "1234567890 QwErTyUiOp ,.(/)&'- name"
+            },
+            "dateOfBirth": "1983-09-24",
+            "countryOfResidence": "DE",
+            "legallyIncapable": false
+          }
+        ]
+      }
+    },
+    "assets": {
```

Comment:
Non-taxable would not have assets in this shape.
They could have a nonEEABusiness